setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal = 1
```

Comment:
Hmm, is that all you need to do for a universal wheel?  I hope it is that easy, that would be great.

Reply:
Yes, as long as:
- runs on Python 2 and 3 with no changes (i.e. it does not require 2to3)
- there are no C extensions.

See https://packaging.python.org/tutorials/distributing-packages/#universal-wheels